aws/gemini-3g/main.tf

```diff
@@ -71,7 +71,7 @@ module "gemini-3g" {
     regions            = var.aws_region
     instance-count     = var.instance_count["domain"]
     docker-org         = "subspace"
-    docker-tag         = "gemini-3g-2023-nov-07"
+    docker-tag         = "gemini-3g-2023-nov-09"
```

Comment:
Wait, why is domain node updated?

Reply:
the changes were not applied on infra, corrected it.